[iOS] Fix issue using TapGesture

jsuarezruiz opened the pull request:
### Description of Change ###

Fix issue using TapGesture on iOS.

### Issues Resolved ### 

- fixes #10623
- fixes #12685
- fixes #13020

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix10623](https://user-images.githubusercontent.com/6755973/87324856-ffdd0b80-c530-11ea-9b0e-c2267a0239ed.gif)


### Testing Procedure ###
Launch Core Gallery and navigate to the issue 10623. Tap the rating control. If you can modify the number of selected stars, the test has passed.

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

samhouts commented:
@jsuarezruiz The test needs to be updated to use brushes.

jsuarezruiz commented:
Updated the test to use brushes.

bholmes commented:
Wow you are kidding me? I do not commit to the repo for years, and one day I decide to pick off something that seems easy. I start to type the commit message and find that you have created this PR. 

Of course I am not really upset, just facepalming at the irony.  

The solution I had was not as good, I disabled user interaction on all the native views in the shape renderer class allowing events to pass through.

bholmes commented:
Here is a copy of my test for this. Verified that the patch passes this test.

https://gist.github.com/bholmes/f7b68c31641e99019c6b11ee6286b575

jsuarezruiz commented:
:( Sorry. Actually this PR was related to 10623 until recently I tried and added 12685 (which is exactly the same).
But, I have added your tests for 12685!

PureWeen commented:
/azp run

rmarinho commented:
Failing test on iOS not related

rmarinho commented on Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue10623.cs:
```diff
@@ -0,0 +1,279 @@
+	{
+		public Issue10623()
+        {
+            Device.SetFlags(new List<string>(Device.Flags ?? new List<string>()) { "Shapes_Experimental" });
```
think we don't need this right?

rmarinho commented:
We should run all uitests as this test is not just on iOS
